src/h5bench.py

```diff
@@ -33,7 +33,7 @@ class H5bench:
     H5BENCH_OPENPMD_READ = 'h5bench_openpmd_read'
     H5BENCH_E3SM = 'h5bench_e3sm'
 
-    def __init__(self, setup, prefix=None, debug=None, abort=None, validate=None):
+    def __init__(self, setup, prefix=None, debug=None, abort=None, validate=None, filter=None):
```

Comment:
Is there any documentation about how to use the filters?

Reply:
Yep, https://h5bench.readthedocs.io/en/filter-benchmarks/running.html#h5bench-recommended